SequenceOfNumbers.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Scanner;
+
+public class SequenceOfNumbers {
+  public static void main(String[] args) {
+    System.out.println("Enter your sequence (use space between numbers):");
+    Scanner in = new Scanner(System.in);
+    String sequence = in.nextLine();
+    String numStr[] = sequence.split(" ");
+    if (numStr.length == 1) {
+      System.out.println("You entered only one number. It is not a sequence");
+      return;
+    }
+    try {
+      for (int i = 0; i + 1 < numStr.length; i++) {
+        if (Integer.parseInt(numStr[i]) > Integer.parseInt(numStr[i+1])) {
+          checkSequence = false;

```

Comment:
where is checkSequence defined? I doubt if this code can be compiled... shame for you